packages/api/controllers/index.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../models');
+const { validate: uuidValidate } = require('uuid');
+
+const getNominationById = async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    if (!uuidValidate(id)) {
+      return res.status(404).send('Nomination with the specified ID does not exist!');
```

Comment:
Nitpicky, but I think it makes sense to change this error to distinguish it a bit from the 404 error below.

Specifically, I think it makes sense to use a "400" error code "Bad Request" and change the message to reflect the fact that the provided UUID is invalid.